### ISSUE TITLE
how to run Armatus using hic file as input

### ISSUE THREAD

**Issue by jiangshan529** [kingsfordgroup/armatus#16]
Hi, dear all,

I am new to use Armutas. I am wondering how to run Armatus using hic file as input. Thanks!

**Comment by yihangs** [comment]
Hi, 

Thank you for your interest. You can read the Armatus README file from GitHub as the starting point, it tells you how to run Armatus from several kinds of hic input files. Feel free to contact us if you have further questions.

**Comment by jiangshan529** [comment]
> Hi,
> 
> Thank you for your interest. You can read the Armatus README file from GitHub as the starting point, it tells you how to run Armatus from several kinds of hic input files. Feel free to contact us if you have further questions.

Hi, Yihang,

Thanks for your reply. Now I have two questions:

1. I downloaded the ubuntu version of Armatus as shown below. When I run it on Linux, it said "armatus: command not found". Does it mean I must install Armatus from source?
```
~/hic/armatus_ubuntu/Armatus-latest_ubuntu-12.04/bin$ ll
total 3880
drwxrwxr-x 2 jib79 jib79    4096 Sep  1  2017 ./
drwxrwxr-x 4 jib79 jib79    4096 Sep  1  2017 ../
-rwxrwxr-x 1 jib79 jib79 3964832 Sep  1  2017 armatus*
jib79@shosty:~/hic/armatus_ubuntu/Armatus-latest_ubuntu-12.04/bin$ armatus
armatus: command not found
```
2. for the input file with `-S` option, now I have a three column sparse matrix file on chr19 like this, is this ok as an input file for Armatus? 
60000   65000   1
60000   70000   1
60000   85000   1
85000   85000   26
85000   90000   16
90000   90000   53
90000   95000   12
95000   95000   27
85000   100000  1
90000   100000  2
95000   100000  10
100000  100000  20
95000   105000  1
100000  105000  14

Thanks!

**Comment by yihangs** [comment]
For the first question: 
You can download via `git clone https://github.com/kingsfordgroup/armatus.git` and go to the directory `./armatus/binaries` to see if you can directly run those executive files, otherwise you may need to compile it by yourself. 
For the second question, 
For the first two columns, you need to divide them by the resolution. 
Hope it helps.

**Comment by jiangshan529** [comment]
> 

Hi, Yihang,

I cloned the git and in the /armatus/binaries directory there are three files:

armatus-linux-x64*
armatus-v2.3-macos-10.12*
armatus-v2.3-ubuntu-x86_64

I just run the file name in linux like this 'armatus-linux-x64', it said "armatus-linux-x64: command not found". Am I using that correctly?

Thanks

**Comment by yihangs** [comment]
try to use `./armatus-linux-x64` instead of `armatus-linux-x64`

**Comment by jiangshan529** [comment]
> try to use `./armatus-linux-x64` instead of `armatus-linux-x64`

Hi, Yihang,

When I run it, it said "./armatus-linux-x64: error while loading shared libraries: libboost_iostreams.so.1.55.0: cannot open shared object file: No such file or directory".
It seems to be Boost problem, I have installed Boost followed this link: https://www.linuxfromscratch.org/blfs/view/7.5/general/boost.html. But the installation is not successful since I am not the sudo user. Do you have any idea to install this?

Thanks.

**Comment by yihangs** [comment]
Yeah installing the libraries is always annoying. I think the best way is to contact the person that helps maintain your cluster to see if he/she can help.
